ee/server/service/maintained_apps.go

```diff
@@ -200,6 +201,18 @@ func (svc *Service) AddFleetMaintainedApp(
 		return 0, ctxerr.Wrap(ctx, err, "creating activity for added software")
 	}
 
+	if automaticInstall && payload.AddedAutomaticInstallPolicy != nil {
+
+		policyAct := fleet.ActivityTypeCreatedPolicy{
+			ID:   payload.AddedAutomaticInstallPolicy.ID,
+			Name: payload.AddedAutomaticInstallPolicy.Name,
+		}
+
+		if err := svc.NewActivity(ctx, authz.UserFromContext(ctx), policyAct); err != nil {
+			return 0, ctxerr.Wrap(ctx, err, "create activity for create automatic install policy for FMA")
```

Comment:
Should probably warn here and continue, because this isn't inside a DB transaction.